generator/v1api.go

```diff
@@ -173,6 +181,14 @@ var (
 		Name:        "stale",
 		In:          inQuery,
 	}
+	tokenAccessorParam = parameter{
```

Comment:
```suggestion
	aclTokenAccessorParam = parameter{
```

Could we prefix this with acl to be consistent with API Area naming convention?

Reply:
Absolutely!